addons/mail/static/src/models/message_view.js

```diff
@@ -317,6 +330,15 @@ registerModel({
                 return true;
             },
         }),
+        hasTextMuted: attr({
+            compute() {
+                if (!this.message) {
+                    return clear();
+                }
+                return this.message.is_note && !this.message.isFromOdooBot;
```

Comment:
Can't we make the message the proper type and/or subtype instead of adding this hacky condition?